src/Combinatorics.jl

```diff
@@ -1,3 +1,5 @@
+__precompile__()
+
 module Combinatorics
 
 using Compat, Polynomials, Iterators
```

Comment:
Compat should still be in REQUIRE if it's still used...

and more specifically relevant to this PR, do both Polynomials and Iterators have `__precompile__` enabled?